### ISSUE TITLE
xUnit.net test runner fails even after uninstalling .Net Core 1.1

### ISSUE THREAD

**Issue by VFSwiss** [xunit/xunit#1104]
Hi.

I tried to upgrade from .net core 1.0 to 1.1, it caused a few issues but the main one was that xunit fails when discovering tests, after trying the solutions I saw here and not getting a fix I decided I could live without 1.1 for now. Unfortunately uninstalling the update made no difference xunit still doesn't work.

It fails with the same error that I got, and other people have been reporting, after the upgrade so I tried the solutions mentoned but with no joy.
Is there anything I can run to get me back to the 1.0 state, could there be a sperarate exe somewhere that is broken by 1.1?
I did a repair on the 003131 sdk and I have restored the nuget packages but that made no difference.

  Thanks, Vincent

So currently I have in global.json
```json
  "sdk": {
    "version": "1.0.0-preview2-003131"
  }
```
In project.json

```json
{
  "version": "1.0.0-*",
  "buildOptions": {
    "debugType": "portable"
  },
  "dependencies": {
    "System.Runtime.Serialization.Primitives": "4.1.1",
    "xunit": "2.1.0",
    "dotnet-test-xunit": "1.0.0-rc2-*",
    "MyProject": "1.0.0-*",
    "Serilog": "2.3.0"
  },
  "testRunner": "xunit",
  "frameworks": {
    "net461": {
      "dependencies": {
        "ServicesWrapper": {
          "target": "project"
        }
      }
    }
  }
}
```
I have tried all the solutions mentioned elsewhere in the forum but nothing helped
`"xunit": "2.2.0-rc1-build3507",`
`"dotnet-test-xunit": "2.2.0-preview2-build1029",`
`"Microsoft.DotNet.InternalAbstractions": "1.0.0"`
and
`"Microsoft.DotNet.InternalAbstractions": "1.0.500-preview2-1-003177"`

**Comment by onovotny** [comment]
xUnit does not support .NET Core App 1.1 due to a breaking change they made with the packages - the one you're hitting.

1.0 should work - can you try to do a clean build - git clean -xdf to ensure it regenerates all of the lock files correctly?

**Comment by VFSwiss** [comment]
Ok, so I will stick with the 1.0 settings that did work before I installed 1.1 then:
`"xunit": "2.1.0"` and `"dotnet-test-xunit": "1.0.0-rc2-*"`

I did the git clean and a dotnet restore but I get the same error.

**Comment by VFSwiss** [comment]
Oops, I accidentally closed this, didn't read the button I was clicking properly :(

You mention the tooling and I forgot to give the details on that.
Not sure if that is relevant but here is what I did with the tooling.

I installed the latest version of the tooling when I installed 1.1 and I then tried to uninstall it when xunit didn't work.
It didn't work cleanly, my "Add/Remove Programs" has entries for both versions, they both say "Microsoft .NET Core 1.0.1 - VS 2015 Tooling Preview 2" but one is version 1.0.20908.53 and the other is 1.0.21114.69. I have tried uninstalls and repairs on both versions without any change.
